tutorials/speaker_tasks/Speaker_Identification_Verification.ipynb

```diff
@@ -1,1267 +1,1228 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "HvYhsOWuSpL_"
+   },
+   "outputs": [],
+   "source": [
+    "trainer.fit(speaker_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "lSRACGt3UAYn"
+   },
+   "source": [
+    "This config is not suited and designed for an4 so you may observe unstable val_loss"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "jvtVKO8FZsoe"
+   },
+   "source": [
+    "If you have a test manifest file, we can easily compute test accuracy by running\n",
+    "<pre><code>trainer.test(speaker_model, ckpt_path=None)\n",
+    "</code></pre>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "FlBwMsRdZfqg"
+   },
+   "source": [
+    "## For Faster Training\n",
+    "We can dramatically improve the time taken to train this model by using Multi GPU training along with Mixed Precision.\n",
+    "\n",
+    "### Trainer with a distributed backend:\n",
+    "<pre><code>trainer = Trainer(devices=2, num_nodes=2, accelerator='gpu', strategy='dp')\n",
+    "</code></pre>\n",
+    "\n",
+    "### Mixed precision:\n",
+    "<pre><code>trainer = Trainer(amp_level='O1', precision=16)\n",
+    "</code></pre>\n",
+    "\n",
+    "Of course, you can combine these flags as well."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "XcnWub9-0TW2"
+   },
+   "source": [
+    "## Saving/Restoring a checkpoint\n",
+    "There are multiple ways to save and load models in NeMo. Since all NeMo models are inherently Lightning Modules, we can use the standard way that PyTorch Lightning saves and restores models.\n",
+    "\n",
+    "NeMo also provides a more advanced model save/restore format, which encapsulates all the parts of the model that are required to restore that model for immediate use.\n",
+    "\n",
+    "In this example, we will explore both ways of saving and restoring models, but we will focus on the PyTorch Lightning method.\n",
+    "\n",
+    "## Saving and Restoring via PyTorch Lightning Checkpoints\n",
+    "When using NeMo for training, it is advisable to utilize the exp_manager framework. It is tasked with handling checkpointing and logging (Tensorboard as well as WandB optionally!), as well as dealing with multi-node and multi-GPU logging.\n",
+    "\n",
+    "Since we utilized the exp_manager framework above, we have access to the directory where the checkpoints exist.\n",
+    "\n",
+    "exp_manager with the default settings will save multiple checkpoints for us -\n",
+    "\n",
+    "1) A few checkpoints from certain steps of training. They will have --val_loss= tags\n",
+    "\n",
+    "2) A checkpoint at the last epoch of training denotes by --last.\n",
```

Comment:
(denote should be passive voice)
A checkpoint at the last epoch of training denotes by --last. ->
Checkpoints at the last epoch of training are denoted by --last.